libs/common/src/sql-db-connection/migrations/1717746522387-PaymentTableUpdateAmountColumns.ts

```diff
@@ -1,69 +1,35 @@
 import { MigrationInterface, QueryRunner } from 'typeorm';
 
-/**
- * This migration has been manually modified in order to work
- * with ledger tables.
- */
 export class PaymentTableUpdateAmountColumns1717746522387 implements MigrationInterface {
   name = 'PaymentTableUpdateAmountColumns1717746522387';
 
   public async up(queryRunner: QueryRunner): Promise<void> {
-    // Create the new columns with the required data-type and set to nullable
     await queryRunner.query(`
-            ALTER TABLE "Payment"
-            ADD "amount" decimal(14, 2) NULL
-        `);
-    await queryRunner.query(`
-            ALTER TABLE "Payment"
-            ADD "reference" nvarchar(255) NULL
+            EXEC sp_rename "[Payment].[amountReceived]", "amount", "COLUMN"
```

Comment:
Twice I've thought about leaving a comment saying can we do this with a new migration, but both times I've gone to delete. I think this is fine to change this one as we can't release. I think we just need to keep this PR open until we know what the next release will be - what we don't want to do is release old code which has this old migration and then we won't run these new changes because the migration has technically been run.
later this week i should know when we can run these scripts and if it can be end of next week